### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by umarcor**
# Bug Fixes

* Fix linter warnings.
* Update Codacy target URLs, from `manual` to `gh`.

### CODE REVIEW

File: pyEDAA/ProjectModel/Xilinx/Vivado.py [hunk 1]
```diff
@@ -149,7 +149,8 @@ def _ParseFile(self, fileNode, fileset):
 		else:
 			self._ParseDefaultFile(fileNode, filePath, fileset)
 
-	def _ParseVHDLFile(self, fileNode, path, fileset):
+	@staticmethod
```
Comment (Paebbels) [inline pyEDAA/ProjectModel/Xilinx/Vivado.py:152]: While it's possible to change it to `staticmethod` I'm not a fan of it for this usecase.

With `staticmethod` these internal helper function get public on the class and can be called from outside standalone, which makes no sense and has other risks.

File: tests/unit/Design.py [hunk 2]
```diff
@@ -140,7 +140,8 @@ def test_Files(self):
 
 
 class Validate(TestCase):
-	def test_Design(self):
+	@staticmethod
+	def test_Design():
```
Comment (Paebbels) [inline tests/unit/Design.py:144]: I think it's not good to convert tests to `staticmethod`s.

Have you checked of pytest is finding these tests?

**Review by Paebbels — CHANGES_REQUESTED** [review]
Maybe we split these fixes from fixes proposed by codacy.

-----------
Maybe we should also limit the `staticmethod` rule in codacy...

**Comment by umarcor** [comment]
I moved commits related to `staticmethod` into #41. Now, this PR contains blank line and list issues only.

**Review by Paebbels — APPROVED** [review]

**Pull request merged**